src/python/dxpy/cli/download.py

```diff
@@ -89,13 +89,14 @@ def download_one_file(project, file_desc, dest_filename, args):
 
 
     try:
+        symlink_max_tries = args.symlink_max_tries if vars(args).get('symlink_max_tries') is not None else 15
         dxpy.download_dxfile(
                             file_desc['id'],
                             dest_filename,
                             show_progress=show_progress,
                             project=project,
                             describe_output=file_desc,
-                            symlink_max_tries=args.symlink_max_tries)
+                            symlink_max_tries=symlink_max_tries)
```

Comment:
Shouldn't it be supplied by the `default` value when args is not provided by the user? https://github.com/dnanexus/dx-toolkit/blob/master/src/python/dxpy/scripts/dx.py#L4576

Reply:
Yes, but this specific `download_one_file()` function is called by `dx get file-xxxx` https://github.com/dnanexus/dx-toolkit/blob/master/src/python/dxpy/scripts/dx.py#L1897 which does not have the `--symlink-max-tries` parameter.

I  added the check here in case `download_one_file()` is called by anything else, although that's doubtful.

Reply:
Thanks!